samples/web/content/apprtc/js/appcontroller.js

```diff
@@ -58,6 +64,8 @@ var AppController = function(loadingParams) {
   this.remoteVideo_ = $(UI_CONSTANTS.remoteVideo);
   this.videosDiv_ = $(UI_CONSTANTS.videosDiv);
   this.roomLinkHref_ = $(UI_CONSTANTS.roomLinkHref);
+  
+  this.inCall_ = false;
```

Comment:
If we need to show self preview before call started, that should be moved out of Call.